### PR TITLE
Tsn163 collect all new tags

### PR DESCRIPTION
**Pull request by sytolk**

### CODE REVIEW

**Comment by sytolk** [comment]
TODO: localization for `i18n.t('core:addTagsToLibrary')`
I see that its have English localization only for General Settings keys. 
Is there a better way to write localization key with default (English) translation in all localization files?

**Comment by uggrock** [comment]
Only the English translation files should be edited. The other files should not have all the keys, because the English is a fallback language. Once a key is translated in Transifex, we have a script which pulls all the changes on a regular basis and integrates them in the repository.

File: app/locales/en/core.json [hunk 1]
```diff
@@ -322,5 +323,6 @@
   "persistIndexSwitch": "Switch to manual index creation with persisted search index",
   "watchForChangesInLocation": "Watch for external changes in this location",
   "proFeature": "feature available in the PRO version",
-  "-": "-"
+  "-": "-",
```
Comment (uggrock) [inline app/locales/en/core.json]: the line with the "-" should be always the last one

File: app/locales/en/core.json [hunk 2]
```diff
@@ -93,6 +93,7 @@
   "useDefaultLocation": "Use default location as a startup location",
   "enablingUsingMetaFolder": "Enable the use of a hidden folder(.ts) for storing meta data and thumbnails",
   "persistTagsInSidecarFile": "Use sidecar files for saving the tags",
+  "addTagsToLibrary": "Add tags to library",
```
Comment (uggrock) [inline app/locales/en/core.json]: Should be renamed to "Collect all new tags to the tag library"

**Pull request merged**